docs/integrations/webhooks/events/index.mdx

```diff
@@ -858,6 +859,28 @@ Payload objects with the following scopes take the form that follows:
 }
 ```
 
+Payload objects with the following scopes take the form that follows:
```

Comment:
@bc-andreadao this ticket has a description of how custom fields webhook works https://bigcommercecloud.atlassian.net/browse/CATALOG-9917 

It was added as one of the `properties` to `store/product/updated` webhook. There's no separate webhook for custom fields. Can you please share which ticket you were using for documentation?

Reply:
@kristinapototska I was using https://bigcommercecloud.atlassian.net/browse/MSF-3602 which talks about `store/product/customFields/updated`. The ticket was also linked to https://bigcommercecloud.atlassian.net/browse/PROJECT-3973

I can take it out though and only document CATALOG-9917

Reply:
@bc-andreadao you can find the example of payload for webhook triggerred after custom fields updates on the screenshot in https://github.com/bigcommerce/webhooks-manager/pull/1173. This is the only supported case for custom fields updates right now

Reply:
@jyuricle Thanks, that URL is giving me a 404 error. It would be great if you could send me the payload screenshot or give me access to the repository

Reply:
<img width="778" alt="Screenshot 2024-07-08 at 18 44 08" src="https://github.com/bigcommerce/docs/assets/14361132/5bd5b244-5f17-41fd-a0e7-000f8319df13">

Reply:
Thanks, it's fixed now